build.sbt

```diff
@@ -37,7 +37,12 @@ addCommandAlias(
   ";coreNative/test:compile"
 )
 
-val zioVersion = "1.0.3"
+val zioVersion = "1.0.4"
+libraryDependencies ++= Seq(
```

Comment:
We shouldn't need to add any `libraryDependencies` here, merely updating `zioVersion` should be enough

Reply:
> We shouldn't need to add any `libraryDependencies` here, merely updating `zioVersion` should be enough

Done! Thanks